easybuild/easyconfigs/i/IMOD/IMOD-4.11.5-foss-2020b.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'ConfigureMake'
+
+name = 'IMOD'
+version = '4.11.5'
+
+homepage = 'https://bio3d.colorado.edu/imod/'
+description = """IMOD is a set of image processing, modeling and display
+programs used for tomographic reconstruction and for 3D reconstruction of EM
+serial sections and optical sections. The package contains tools for assembling
+and aligning data within multiple types and sizes of image stacks, viewing 3-D
+data from any orientation, and modeling and display of the image files. IMOD
+was developed primarily by David Mastronarde, Rick Gaudette, Sue Held, Jim
+Kremer, Quanren Xiong, and John Heumann at the University of Colorado."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['5c50f1457134fe5113b95e0939d3181b948d211f25d9a05ad2b4ec0f23251c79']
+
+# mercurial repository
+# hg clone --debug http://bio3d.colorado.edu/imod/nightlyBuilds/IMOD
+# get lunch 
+# cd IMOD
+# hg update -r IMOD_4-11-5
+# cd ..
+# tar czf IMOD-4.11.5.tar.gz IMOD
+
+
+dependencies = [
+    ('LibTIFF', '4.1.0'),
+    ('Qt5', '5.14.2'),
+    ('Java', '11', '', True),
+    ('HDF5', '1.10.7'),
+    ('Python', '3.8.6'),
+]
+
+# modify all qmake pro files in order to pass CFLAGS
+local_qmake_pass_cflags = "find -name *.pro -exec sed -i -e '$aQMAKE_CXXFLAGS += $$(CFLAGS)' {} \\; &&"
+
+# exports required for configure and build
+local_exports = 'export QTDIR=$EBROOTQT5 &&'
+local_exports += 'export HDF5_DIR=$EBROOTHDF5 &&'
+local_exports += 'export QMAKESPEC=$EBROOTQT5/mkspecs/`qmake -query QMAKE_SPEC` &&'
+#  readw_or_imod.f and others with gfortran10: 
+#     Error: Type mismatch between actual argument at (1) and actual argument at (2) (INTEGER(4)/INTEGER(2))
+#     => set -fallow-argument-mismatch. Runs through without this option with GCC 8.3.0.
+local_exports += 'export CFLAGS="$CFLAGS -fallow-argument-mismatch" &&'  # required for gfortran10
+
+preconfigopts = local_exports
+preconfigopts += local_qmake_pass_cflags
+
+# IMOD's configure script is named setup and does not know the parameter --prefix, but -i. 
+# CFLAGs are passed with -flags.
+configure_cmd = './setup '
+configure_cmd += '-flags "$CFLAGS" '    # inject CFLAGS
+configure_cmd += '-i %(installdir)s '  # set installdir
+configure_cmd += ' #'  # avoid passing unknown arg --prefix
+
+prebuildopts = local_exports
+
+# create some missing directories required for installation process:
+preinstallopts = 'mkdir %(installdir)s/{man/cat1,bin,autodoc,SystemTemplate,lib/imodplug,com,html,Plugins} -p &&'
+
+modextravars = {
+    'IMOD_DIR': "%(installdir)s",
+    'IMOD_PLUGIN_DIR': '%(installdir)s/lib/imodplug',
+    'IMOD_JAVADIR': '$JAVA_HOME',
+    'FOR_DISABLE_STACK_TRACE': '1',
+}
+modloadmsg = 'Please set the environment variable $IMOD_CALIB_DIR if appropriate.'
+# Observed problems with alias on a tcl based module system. Use bin/subm python script instead.
+# modaliases = {
```

Comment:
why did you comment it out? is there a downside to enabling this by default?

Reply:
I decided, not to enable the alias in order to make the EC more universal. As there is also a subm python script avialable, the alias is not mandatory.